src/Illuminate/Contracts/Foundation/Application.php

```diff
@@ -237,6 +237,11 @@ public function setLocale($locale);
      */
     public function shouldSkipMiddleware();
 
+    /**
+     * @return bool
+     */
+    public function shouldMergeBaseConfiguration();
```

Comment:
We can't add methods to a contract on a patch release.

Reply:
I removed the method from interface and added `method_exists($app, 'shouldMergeBaseConfiguration')` check where the method is used. Would that work?